STYLE.md

```diff
@@ -0,0 +1,147 @@
+# Documentation Style and Formatting
+
+## English Style
+
+Write short sentences. Organize concepts in paragraphs. Prefer lists to tables and paragraphs to lists. Write in the active voice. Avoid jargon beyond the requirements of subject and audience.
+
+### Eschew You
+
+You write unambiguous documentation, so you avoid the second person. Avoiding personal pronouns in general helps produce the imperative impersonal tone desired for documentation. Don't reboot your system or have the user reboot their system. Reboot the system.
+
+### Generalities
+
+There are a few other common ways to write or not write things:
+
+* Expand acronyms on their introduction in a document, with the short form following in parentheses: Trusted Platform Module (TPM)
+* Terms of art that are not commands or other literal text should often be italicized on their first appearance in a document: *Kubernetes* is a good example
+* The hyphen is overused and most English compounds do not require it
+
+## Source Formatting
+
+CoreOS documentation is written in [Markdown](https://daringfireball.net/projects/markdown/syntax), a simple way to annotate text to indicate presentation typesetting. Markdown source is intended to be a plain text human-readable version of the document, even before conversion to HTML for the browser or other display.
+
+### File Naming and Encoding
+
+Write Markdown source in UTF-encoded plain text files, named with a reasonable, lower-case short form of the document's title, and suffixed with `.md`. Prefer hyphens to underscores in multi-word file names. For example, instructions for DNS configuration are written to a file named [`configuring-dns.md`](os/configuring-dns.md).
+
+### Line Wrapping Considered Harmful
+
+Don't wrap long lines of text with manual newlines. Line wrapping churns prose documents, because lines not actually edited will nevertheless change when a paragraph is edited and re-wrapped.
+
+### One Sentence per Line Deprecated
+
+Writing Markdown source with a newline between every sentence is acceptable to most compilers and can ease change review. However, this format makes the document less readable in source form. Do not add a line break between sentences. Write natural English paragraphs, separated by two newlines.
+
+### Preferred Markdown Symbols
+
+Markdown defines two or more ways to declare some document structures. This documentation prefers these Markdown symbols among their alternatives:
+
+* Headings are denoted in Markdown's ATX style, with hash character(s): `#`. See [*Headings*](#headings), below.
+* Bulleted lists, like this one, are denoted with the asterisk (`*`), rather than the hyphen.
+* Hyperlink URLs are given inline (`[hyperlinked text](target.md)`), rather than reference-style. Relative links are preferred to absolute links. See [*Hyperlink Considerations*](#hyperlink-considerations), below.
```

Comment:
I personally prefer reference-style as well. Mainly because I find it more readable. Otherwise, I have to train my eyes to recognize and skip over URLs.